homeassistant/components/govee_ble/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The Govee Bluetooth BLE integration."""
+from __future__ import annotations
+
+import logging
+
+from govee_ble import GoveeBluetoothDeviceData
+
+from homeassistant.components.bluetooth.passive_update_coordinator import (
+    PassiveBluetoothDataUpdate,
+    PassiveBluetoothDataUpdateCoordinator,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.service_info.bluetooth import BluetoothServiceInfo
+
+from .const import DOMAIN
+from .sensor import sensor_update_to_bluetooth_data_update
```

Comment:
I realize now that we've gone against a design principle that we normally follow: platform details should be kept with the platform. If we ever want to support more than a single platform with the coordinator, the current design doesn't work with that.

It's not flexible that the coordinator handles entity details, like entity descriptions and names.

Reply:
I think it could be possible that passive data could be a binary sensor as well (haven't found one yet though). 

Will think about how to refactor to handle that

Reply:
I pushed up a prospective change to handle multiple platforms.  It really doesn't belong in this PR, but it seems liked the easiest way to  get feedback, and I'll pull it out after we decide on a direction.

Reply:
https://github.com/home-assistant/core/pull/75642